lib/src/posthog_web.dart

```diff
@@ -20,7 +19,7 @@ class PosthogWeb {
     switch (call.method) {
       case 'identify':
         analytics.callMethod('identify', [
-          call.arguments['distinctId'],
```

Comment:
I think keeping it as `distinctId` keeps it aligned with our JS library
https://github.com/PostHog/posthog-js/blob/master/src/posthog-core.js#L810

Have you tried passing your `userId` as the `distinctId` parameter?

Reply:
Thank you for giving me a feedback!
I changed distinctId to userId because of the default setting of Posthog().identify.
To use  Posthog().identify, there is a required parameter "userId".
So I have to use like this. 
```
Posthog().identify(userId: "Shinyoung Tyga Lee");
```

Reply:
@fuziontech 
Thank you for giving me a feedback!
I changed distinctId to userId because of the default setting of Posthog().identify.
To use  Posthog().identify, there is a required parameter "userId".
So I have to use like this.  I think that using distinctId should make change about default setting of required parameter "userId" name.
```
Posthog().identify(userId: "Shinyoung Tyga Lee");
```

Reply:
You are totally right! https://github.com/PostHog/posthog-flutter/blob/7beff569182860749afcd802d9d8155c464b4ed0/lib/src/posthog.dart#L19
Your changes look great and make this more consistent.